website/content/docs/concepts/node-pools.mdx

```diff
@@ -0,0 +1,366 @@
+creates the node pool automatically on client registration.
+
+Jobs can then reference node pools using the [`node_pool`][job_np] attribute.
+
+```hcl
+job "app-dev" {
+  # ...
+  node_pool = "dev"
+  # ...
+}
+```
+
+Similarly to the `namespace` attribute, the node pool must exist beforehand,
+otherwise the job registration results in an error. Only nodes in the given
+node pool are considered for placement. If none are available the deployment
+is kept as pending until a client is added to the node pool.
+
+## Multi-region Clusters
+
+In federated multi-region clusters, node pools are automatically replicated
+from the authoritative region to all non-authoritative regions, and requests to
+create or modify a new node pool are forwarded from non-authoritative to the
+authoritative region.
+
+Since the replication data only flows in one direction, clients in
+non-authoritative regions are not able to create node pools on demand.
+
+A client in a non-authoritative region that references a node pool that does
+not exist yet is kept in the `initializing` status until the node pool is
+created and replicated to all regions.
+
+## Built-in Node Pools
+
+In addition to the user generated node pools Nomad automatically creates two
+built-in node pools that cannot be deleted nor modified.
+
+- `default`: Node pools are an optional feature of Nomad. The `node_pool`
+  attribute in both the client configuration and job files are optional. When
+  not specified, these values are set to use the `default` built-in node pool.
+
+- `all`: In some situations, it is useful to be able to run a job across all
+  clients in a cluster, regardless of their node pool configuration. For these
+  scenarios the job may use the built-in `all` node pool which always includes
+  all clients registered in the cluster. Unlike other node pools, the `all`
+  node pool can only be used in jobs and not in client configuration.
+
+## Nomad Enterprise <EnterpriseAlert inline />
+
+Nomad Enterprise provides additional features that make node pools more
+powerful and easier to manage.
+
+### Scheduler Configuration
+
+Node pools in Nomad Enterprise are able to customize some aspects of the Nomad
+scheduler and override certain global configuration per node pool.
+
+This allows experimenting with with functionalities such as memory
+oversubscription in isolation, or adjusting the scheduler algorithm between
+`spread` or `binpacking` depending on the types of workload being deployed in a
+given set of clients.
+
+When using the built-in `all` node pool the global scheduler configuration is
+applied.
+
+Refer to the [`scheduler_config`][np_spec_scheduler_config] parameter in the
+node pool specification for more information.
+
+### Node Pool Governance
+
+Node pools and namespaces share some similarities, with both providing a way to
+group resources in isolated logical units. Jobs are grouped into namespaces and
+clients into node pools.
```

Comment:
This phrase is very clear and useful, I would add it in the node pools description

Reply:
Thanks! Which node pool description are you referring to?